### ISSUE TITLE
Where can I find the test set you used?

### ISSUE THREAD

**Issue by Sopralapanca** [Tool-LMM/Tool-LMM#7]
The dataset provided seems to contain only training data. Where can I find the test set data?

**Comment by Chenyu-Wang567** [comment]
Thanks a lot for your interest in our work! 
The test set is available under the data/ directory. You can check the specific JSON files referenced in inference.py
, Line 152–153. For multimodal data refered in the file, please download it from the provided Google Drive link.

If you have any further questions, feel free to open another issue or continue the discussion here.
